pkg/rbd/rbd_util.go

```diff
@@ -96,10 +96,10 @@ var (
 	volumeNameLocker = util.NewIDLocker()
 	// serializes operations based on "snapshot name" as key
 	snapshotNameLocker = util.NewIDLocker()
```

Comment:
`attachdetachLocker` above can be removed, as this locks the imageName appended with the Pool name, and will be handled by the `nodeVolumeIDLocker` lock.